examples/dao/test/dao-app-flow.js

```diff
@@ -530,10 +529,11 @@ describe("DAO test", function () {
 		syncAccounts();
 
 		// verify proposalALsig recieved back deposit of 15 tokens
```

Comment:
this comment description should be changed to follow your changes code

Reply:
done